keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2619,6 +2619,14 @@ Beta:
   - Automatically manages versions of mandatory/suggested configuration
   - Discrimates paginated LIST requests
 
+GA:
+
+- Satisfaction with LIST and WATCH support
+- APF allows us to disable client-side rate limiting (or we know the reason why not)
+- Satisfaction that the API can be extended in a backward-compatible way to support borrowing between priority levels
```

Comment:
I'm not clear on why we would consider the feature GA without borrowing being implemented.

Reply:
We all are using P&F in production now and we've all seen cases where it actually prevented failover. So we have proofs that it already provides significant value even without borrowing.
In my personal opinion borrowing is an extensions/feature on top of the basic P&F and (while I fully agree that we should start working on that now-ish), I don't think we should actually block the GA of the feature as a whole on it.
WDYT?

Reply:
Without borrowing, we have reservations about really compressing the number of concurrent requests to a value small enough to keep clusters near the edge.  I'd like to be able to shrink the number of concurrent requests, but that has significant negative impacts without borrowing unused priority.

cc @tkashem

Reply:
I fully agree with the above. And I 100% agree we should do that.
My only concern is - why this should be a GA release blocker. All of us are effectively using P&F in production. And I think at this point no-one will switch back to max-in-flight as it's already visible worse.
So the way I'm looking at it is "P&F is missing an important extension", not "P&F is not GA-quality".

Reply:
We discussed it in the APF meeting today and decided to update the criteria to say borrowing is working.